.goreleaser.yml

```diff
@@ -121,7 +121,7 @@ builds:
     goamd64:
       - v1
     prebuilt:
-      path: dist/confluent_{{ .Os }}_{{ .Arch }}_{{ with .Amd64 }}_v1{{ end }}/confluent
+      path: dist/confluent_{{ .Os }}_{{ .Arch }}_{{ with .Amd64 }}_{{ . }}{{ end }}/confluent
```

Comment:
Is the `.` a wildcard?

Reply:
I think if `.Amd64` is defined, the `.` gets replaced with its value. I'm copying from the documentation here: https://goreleaser.com/customization/builds/#import-pre-built-binaries (Although I couldn't find any documentation that explains how it works... so I'm going in blind)